clients/python/poetry.lock

```diff
@@ -971,43 +990,40 @@ zstd = ["zstandard (>=0.18.0)"]
 
 [[package]]
 name = "huggingface-hub"
-version = "0.35.3"
+version = "1.0.1"
```

Comment:
we use HF-hub for "import from HF" flow: https://model-registry.readthedocs.io/en/latest/#importing-from-hugging-face-hub 

a major version upgrade

/lgtm
/approve